packages/database/src/browser/MigrationManager.js

```diff
@@ -0,0 +1,145 @@
+import { PGlite } from '@electric-sql/pglite';
+import path from 'path';
+import { getConnectionConfig } from './getConnectionConfig';
+
+const DEFAULT_MIGRATIONS_PATH = path.resolve(__dirname, '/packages/database/src/core/migrations');
+
+export class MigrationManager {
```

Comment:
Out of scope for this ticket:
- Running the migration manager on start up (we probably want to run this when the browser app is updated to a newer version?), and that will be done in another ticket, so it's not being executed yet, but it's all ready here.